js_modules/dagster-ui/packages/ui-core/src/runs/flattenNodeChunks.tsx

```diff
@@ -0,0 +1,6 @@
+import {LogNode} from './types';
+
+// https://jsbench.me/o8kqzo8olz/1
+export function flattenNodeChunks(nodeChunks: LogNode[][]) {
+  return ([] as LogNode[]).concat(...nodeChunks);
+}
```

Comment:
Chunk flattener, based on benchmark noted in comment.

Reply:
Can we use a generic type parameter to make this utility work with more than just `LogNode`?